Examples/Swift/CustomViewController.swift

```diff
@@ -65,7 +64,6 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
     }
 
     func suspendNotifications() {
-        NotificationCenter.default.removeObserver(self, name: .routeControllerDidPassSpokenInstructionPoint, object: nil)
```

Comment:
Should we pass the instructions to the MapboxVoiceController?

Reply:
No need to. When MapboxVoiceController is inited, it starts listening on it's own.